cumulusci/cli/cli.py

```diff
@@ -591,16 +596,32 @@ def org_info(config, org_name):
 def org_list(config):
     check_connected_app(config)
     data = []
-    headers = ['org', 'is_default']
+    headers = ['org', 'default', 'scratch', 'config_name', 'username']
     for org in config.project_config.list_orgs():
         org_config = config.project_config.get_org(org)
-        if org_config.default:
-            data.append((org, '*'))
-        else:
-            data.append((org, ''))
+        row = [org]
+        row.append('*' if org_config.default else '')
+        row.append('*' if org_config.scratch else '')
+        row.append(org_config.config_name if org_config.config_name else '')
+        username = org_config.config.get('username')
+        if not username:
+            username = org_config.userinfo__preferred_username
+        row.append(username if username else '')
+        data.append(row)
     table = Table(data, headers)
     click.echo(table)
 
+@click.command(name='remove', help="Removes an org from the keychain")
+@click.argument('org_name')
+@click.option('--global-org', is_flag=True, help="Set this option to force remove a global org.  Default behavior is to error if you attempt to delete a global org.")
+@pass_config
+def org_remove(config, org_name, global_org):
+    check_connected_app(config)
+    try:
+        org_config = config.keychain.get_org(org_name)
```

Comment:
Unused output var?

Reply:
I'm confused on this one... do you mean global_org isn't used?

Reply:
org_config isn't used :)